OTEP.md

```diff
@@ -17,26 +19,38 @@ The environment variable interface suffers from the following identified limitat
 * **Limited validation**. Validation can only be performed by the receiver, there is no meta-configuration language to validate input.
 * **Difficult to extend**. It’s difficult to anticipate the requirements of configuring custom extension components (processors, exporters, samplers, etc), and likely not practical to represent them in a flat structure. As a result, the environment variable interface is limited to components explicitly mentioned in the specification.
 
-## Explanation (TBD)
+## Explanation
 
-Using a configuration model or configuration file, users could configure all options currently available via environment variables.
+Using a configuration model or configuration file, users can configure all options currently available via environment variables.
 
 ### Goals
```

Comment:
We may want to add a goal to allow some sort of variable substitution in the config file. Layering environment variables into config files (as is done in the collector today) is pretty important to avoid plain text secrets.

Reply:
Done